WordPress/Classes/ViewRelated/Notifications/Style/WPStyleGuide+Notifications.swift

```diff
@@ -229,9 +226,6 @@ extension WPStyleGuide {
         fileprivate static let snippetParagraph         = NSMutableParagraphStyle(
             minLineHeight: snippetLineSize, maxLineHeight: snippetLineSize, lineBreakMode: .byWordWrapping, alignment: .natural
```

Comment:
I do know that sometimes, with `UITableViewAutomaticDimension`, the calculated height might be fractional and break the lines incorrectly when using tail truncation. If that rings a bell, @jleandroperez